README.md

```diff
@@ -75,7 +94,7 @@ Making the `<script>` asynchronous is a highly recommend because it reduces the
 ### Alternative: Use the built in PayPalSDKWrapper component
 
 ```javascript
-import SmartPaymentButtons, { PayPalSDKWrapper } from 'react-smart-payment-buttons';
+import PayPalSDKWrapper, { SmartPaymentButtons, SmartSubscriptionButtons } from 'react-smart-payment-buttons';
```

Comment:
Yeah, see above. The `PayPalSDKWrapper` also is not really a default export for me because it's rather additional.